src/OrchardCore.Modules/OrchardCore.Localization/Permissions.cs

```diff
@@ -0,0 +1,35 @@
+using System.Collections.Generic;
+using OrchardCore.Security.Permissions;
+
+namespace OrchardCore.Localization
+{
+    public class Permissions : IPermissionProvider
+    {
+        public static readonly Permission ManageCultures = new Permission("ManageCultures", "Manage supported culture");
+
+        public IEnumerable<Permission> GetPermissions()
+        {
+            return new[] 
```

Comment:
Can we use `yield return ManageCultures` instead of creating an array each time?